README.md

```diff
@@ -2,6 +2,9 @@
 
 A simple game inspired by Age of Castles. Distribute your population to farming, construction, guard duty and scholarship, survive demon invasion and find the way to banish them forever.
 
+<a href="https://f-droid.org/packages/hr.kravarscan.enchantedfortress/" target="_blank">
```

Comment:
I haven't seen other readme so far, that use `target="_blank"`, but otherwise LGTM.

Reply:
I can remove it if you want?

Reply:
Yes, that would be good.

Reply:
Done :)